vllm/executor/multiproc_gpu_executor.py

```diff
@@ -218,7 +218,7 @@ def __init__(self, *args, **kwargs):
 
     async def _driver_execute_model_async(
         self,
-        execute_model_req: Optional[ExecuteModelRequest] = None
```

Comment:
Just a nit but in general it's best to do unrelated formatting fixes in a separate PR so it's easier to see the "real" changes corresponding to the thing the PR is implementing.

Reply:
fixed, thanks for suggesting!